repos/spack_repo/builtin/packages/py_scifem/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyScifem(PythonPackage):
+    """A collection of tools for scientific computing with a focus on finite element methods.
+    The tools are written in Python and are intended to be used in conjunction with
+    the py-fenics-dolfinx.
+    """
+
+    homepage = "https://scientificcomputing.github.io/scifem/"
+    url = "https://github.com/scientificcomputing/scifem/archive/refs/tags/v0.6.0.tar.gz"
+    git = "https://github.com/scientificcomputing/scifem.git"
+
+    maintainers("finsberg", "jorgensd")
```

Comment:
@finsberg Are you fine getting pinged for changes to this package recipe?

Reply:
Yes, I am fine with that 👍